### PR TITLE
Configures CircleCI.

### PR DESCRIPTION
**Pull request by scherztc**
Fixes Issue #739 
Fixes issue #494

Uses Samvera ORB
Pins Puma gem
Pins Rake gem

### CODE REVIEW

**Comment by scherztc** [comment]
We are green again and coverage is updated, BUT still low.  I wonder if this number represents only job 141.2

Screen Shot 2020-07-07 at 5 32 38 PM

**Comment by scherztc** [comment]
After using explicit rspec commands instead of rspec_parallel from Samvera

Screen Shot 2020-07-08 at 12 35 20 AM

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+#                  bundle exec rspec \
+#                    --format progress \
+#                    --out /tmp/test-results/rspec.xml \
+#                    --format progress \
+#                  $TEST_FILES
+
+            - samvera/parallel_rspec
+
+            - store_artifacts:
+                path: tmp/screenshots
```
Comment (hortongn) [inline .circleci/config.yml:94]: @scherztc See the screenshot below.  I looked closer at the circleci interface and can see the total number fo tests and how to view each parallel build.

build

I added the coveralls token to the circleci account and it likes it.  But I think we have to change `path: tmp/screenshots` to `path: coverage` to get the coveralls integration working properly.  So try that change and clean up the rest of the file so we can merge.  If we get coveralls working here we can try the same on our other rails projects.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Scholar@UC
 ![Travis (.org) branch](https://img.shields.io/travis/uclibs/ucrate/develop.svg) ![Coveralls coverage](https://img.shields.io/coveralls/github/uclibs/ucrate/develop.svg)
+[![CircleCI](https://circleci.com/gh/uclibs/ucrate.svg?style=svg)](https://circleci.com/gh/uclibs/ucrate)
+[![Coverage Status](https://coveralls.io/repos/github/uclibs/ucrate/badge.svg?branch=master)](https://coveralls.io/github/uclibs/ucrate?branch=master)
```
Comment (hortongn) [inline README.md:4]: Also be sure to update the README docs to use 8080 port for fedora testing

**Pull request merged**